datafusion-expr/src/built_in_function.rs

```diff
@@ -0,0 +1,330 @@
+// Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
is the plan to move the built in aggregate functions here too?

Reply:
i think it's in the `aggregate_function` module